### ISSUE TITLE
Streams not working in VS Code or holoviews==1.16.0a8 in Jupyter notebook

### ISSUE THREAD

**Issue by basnijholt** [holoviz/holoviews#5711]
#### ALL software version info
Tested in VS Code
```
import holoviews, bokeh
holoviews.__version__, bokeh.__version__
('1.15.4', '2.4.3')
```
Tested in Jupyter notebook:
```
('1.16.0a8', '3.1.1')
```
#### Complete, minimal, self-contained example code that reproduces the issue
```python
def gen():
    while True:
        ys = np.random.rand(10)
        yield hv.Curve(ys)

dm = hv.DynamicMap(gen(), streams=[hv.streams.Stream.define("Next")()])
dm
```

next cell 
```
dm.periodic(0.5)  #next cell
```

#### Screenshots or screencasts of the bug in action

![dm](https://user-images.githubusercontent.com/6897215/236653827-0ab0d329-4690-44d5-9bd8-ff8a1abc33a3.gif)

# Edit: in a Jupyter notebook and the most recent alpha release 1.16.0a8, this doesn't work either.

**Comment by basnijholt** [comment]
@Hoxbro, I also noticed that this doesn't work at all in a notebook Jupyter notebook and holoviews==1.16.0a8.

**Comment by Hoxbro** [comment]
It seems to work fine for me in Jupyter Lab, Jupyter Notebook, and VS Code. 

``` python
import bokeh
import holoviews as hv
import numpy as np
import panel as pn

hv.extension("bokeh")

hv.__version__, bokeh.__version__, pn.__version__

def gen():
    while True:
        ys = np.random.rand(10)
        yield hv.Curve(ys)


dm = hv.DynamicMap(gen(), streams=[hv.streams.Stream.define("Next")()])
dm

dm.periodic(0.5)  # next cell
```


https://user-images.githubusercontent.com/19758978/236666512-05c9dd1e-d1b2-4375-8fdd-e88a74d5393f.mp4

**Comment by basnijholt** [comment]
I just retried and it seemed like I was missing `jupyter_bokeh`. I can now confirm that Jupyter notebook and VS Code both work with 1.16.0a8.

However, I really cannot get it to work with the most recently released version of HoloViews (1.15.4). I tried it in a fresh devcontainer in VS Code and get the same behavior. Since it seems like this might be fixed in the upcoming release, I will just close this and be patient :) 

Thanks for your reply!
